recipe/recipe.yaml

```
@@ -98,71 +99,80 @@ requirements:
   - aiohttp
   - blake3
   - cachetools
+  - cbor2
   - cloudpickle
-  - compressed-tensors ==0.9.2
-  - depyf ==0.18.0
+  - compressed-tensors ==0.10.2
+  - depyf ==0.19.0
+  - diskcache ==5.6.3
   - einops
   - fastapi >=0.115.0
   - filelock >=3.16.1
-  - gguf ==0.10.0
+  - gguf ==0.13.0
   - importlib-metadata
-  - hf-xet >=0.1.4
-  - huggingface_hub >=0.30.0
+  - hf-xet >=1.1.2,<2.0.0
+  - huggingface_hub >=0.33.0
   - lark ==1.2.2
   - lm-format-enforcer >=0.10.11,<0.11
-  - mistral-common >=1.5.4
+  - mistral-common >=1.8.2
   - msgspec
+  # Numba v0.61 doesn't support Python 3.9. Required for N-gram speculative decoding
+  # Requirements taken from https://github.com/vllm-project/vllm/blob/296c6572dd1f76b31b93be19e550790afcfb8843/requirements/cuda.txt#L4-L5
+  # and https://github.com/vllm-project/vllm/blob/6d8d0a24c02bfd84d46b3016b865a44f048ae84b/requirements/cpu.txt#L4-L5
+  # Also, because conda-forge's torchaudio dropped support for Python 3.9, we don't include the conditional numba 0.60 run requirement.
+  - numba ==0.61.2
   - numpy
-  - openai >=1.52.0
+  - openai >=1.87.0,<= 1.90.0
   - opencv >=4.11.0
-  - outlines ==0.1.11
+  - outlines ==0.2.10
   - partial-json-parser
   - pillow
   - prometheus_client >=0.18.0
   - prometheus-fastapi-instrumentator >=7.0.0
   - protobuf
   - psutil
   - py-cpuinfo
-  - pydantic >=2.9
+  - pybase64
+  - pydantic >=2.10
   - python-json-logger
   - pytorch ==${{ pytorch_version }}
   - pyyaml
   - pyzmq
+  - regex
   - requests >=2.26.0
   - scipy
   - sentencepiece
   - tiktoken >=0.6.0
-  - tokenizers >=0.19.1
+  - tokenizers >=0.21.1
   - tqdm
-  - transformers >=4.51.0
+  - transformers >=4.53.2
   - typing_extensions >=4.10
   - uvicorn-standard
   - watchfiles
   - if: x86_64 or arm64 or aarch64
     then:
-    - llguidance >=0.7.9,< 0.8.0
-  - if: x86_64 or aarch64
-    then:
-    - xgrammar ==0.1.17
+    - llguidance >=0.7.11,< 0.8.0
+    - xgrammar ==0.1.21
   - if: match(python, ">3.11")
     then:
     - six >=1.16.0
-    - setuptools >=74.1.1
+    - setuptools >=77.0.3,<80
   - if: use_cuda
     then:
-    # Numba v0.61 doesn't support Python 3.9. Required for N-gram speculative decoding
-    # Requirements taken from https://github.com/vllm-project/vllm/blob/296c6572dd1f76b31b93be19e550790afcfb8843/requirements/cuda.txt#L4-L5
-    # Also, because conda-forge's torchaudio dropped support for Python 3.9, we don't include the conditional numba 0.60 run requirement.
-    - numba ==0.61
     - ray-cgraph >=2.43.0,!=2.44
     - torchaudio ==${{ pytorch_version }}
-    - torchvision ==0.21.0
+    - torchvision ==0.22.1
     - if: linux64
       then:
-      - xformers ==0.0.29.post2  # platform_system == "Linux" and platform_machine == "x86_64"
+      - xformers ==0.0.31  # platform_system == "Linux" and platform_machine == "x86_64"
```

```suggestion
      - xformers ==0.0.31.*  # platform_system == "Linux" and platform_machine == "x86_64"
```
If you want to allow the `.post1` builds, which should be compatible